profiles/net_4_x/System.Net.Http.WebRequest.cs

```diff
@@ -61,13 +61,12 @@ public partial class WebRequestHandler : System.Net.Http.HttpClientHandler
         public bool AllowPipelining { get { throw null; } set { } }
         public System.Net.Security.AuthenticationLevel AuthenticationLevel { get { throw null; } set { } }
         public System.Net.Cache.RequestCachePolicy CachePolicy { get { throw null; } set { } }
-        public new System.Security.Cryptography.X509Certificates.X509CertificateCollection ClientCertificates { get { throw null; } }
         [System.MonoTODOAttribute]
         public System.TimeSpan ContinueTimeout { get { throw null; } set { } }
         public System.Security.Principal.TokenImpersonationLevel ImpersonationLevel { get { throw null; } set { } }
-        public new int MaxResponseHeadersLength { get { throw null; } set { } }
```

Comment:
I could be wrong, but these two APIs are still present in .NET Framework 4.7.2. Wouldn't it cause errors for code that was compiled against the framework and uses these APIs? If yes, can you add these properties back and implement them by calling the `base.` properties that they override?

Reply:
I gave it a try and removing the "new" method actually works, so you can disregard the comment.